packages/web/src/components/usdc-balance-pill/UsdcBalancePill.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  BNUSDC,
+  Status,
+  formatCurrencyBalance,
+  formatUSDCWeiToFloorCentsNumber,
+  useUSDCBalance
+} from '@audius/common'
+import { LogoUSDC } from '@audius/stems'
+import BN from 'bn.js'
+import cn from 'classnames'
+
+import { Icon } from 'components/Icon'
+import Skeleton from 'components/skeleton/Skeleton'
+
+import styles from './UsdcBalancePill.module.css'
+
+type USDCPillProps = {
+  className?: string
+}
+
+export const UsdcBalancePill = ({ className }: USDCPillProps) => {
+  const { data: balance, balanceStatus: usdcBalanceStatus } = useUSDCBalance({
+    isPolling: false
+  })
+  const isLoading = balance === null || usdcBalanceStatus === Status.LOADING
+  const balanceCents = formatUSDCWeiToFloorCentsNumber(
```

Comment:
might be worth trying the `USDC(balance)` w/ `toLocaleString()` here, lmk what you think